diff --git a/dist/server/lib/revalidate.js b/dist/server/lib/revalidate.js
index c94bc3b53e748f90f557be47fa7854a7919d9c07..87248a5561c68b63c5e0afdefca4769679d96a1c 100644
--- a/dist/server/lib/revalidate.js
+++ b/dist/server/lib/revalidate.js
@@ -11,7 +11,7 @@ Object.defineProperty(exports, "formatRevalidate", {
 const _constants = require("../../lib/constants");
 function formatRevalidate(revalidate) {
     if (revalidate === 0) {
-        return "private, no-cache, no-store, max-age=0, must-revalidate";
+        return "public, max-age=0, must-revalidate";
     } else if (typeof revalidate === "number") {
         return `s-maxage=${revalidate}, stale-while-revalidate`;
     }